addon/components/sl-select.js

```diff
@@ -240,7 +240,7 @@ export default Ember.Component.extend( InputBased, TooltipEnabled, {
                 query: ( query ) => {
                     const content = this.get( 'content' ) || [];
                     const optionLabelPath = this.get( 'optionLabelPath' );
-                    const select2 = this;
+                    const select2 = input.data( 'select2' ).opts;
```

Comment:
What effect does this change have?  Is it related to https://github.com/softlayer/sl-ember-components/issues/433?